Make React Router more visible (#3531)
getsentry/sentry-docs: src/platforms/common/performance/index.mdx

</PlatformSection>

<!-- notSupported here should be the opposite of whats in common/connect-services.mdx -->

<PlatformSection notSupported={["javascript", "javascript.cordova"]}>

## Connecting Services

If you are also using Performance Monitoring for [JavaScript](/platforms/javascript/performance/), depending on where your request originates, you can connect traces:

1. For requests that start in your backend, by [adding a meta tag](/platforms/javascript/performance/connect-services/#pageload) in your HTML template that contains tracing information.
2. For requests that start in JavaScript, by the SDK [setting a header](/platforms/javascript/performance/connect-services/#navigation-and-other-xhr-requests) on requests to your backend.

Otherwise, backend services with Performance Monitoring connect automatically.

</PlatformSection>

> <PlatformSection supported={["javascript",]} notSupported={["react-native"]}>

Reviewer: ```suggestion
<PlatformSection supported={["javascript.react"]}>
```

We should scope this to react only. (That should also fix all of those broken links above.)

## React Router

> If you are using `react-router`, we provide `react-router` instrumentation you can use alongside Performance Monitoring with `@sentry/tracing`. Please see the [React Router Integration documentation](configuration/integrations/react-router/) for more details on setting it up.

Reviewer 2: Editing to make this fewer words and increase readability
```suggestion
If you are using `react-router`, we provide `[react-router` instrumentation](configuration/integrations/react-router/) to use with our `@sentry/tracing` package. 
```
